docs/blocks/index.md

```diff
@@ -54,7 +54,7 @@ For simple blocks which need no special functionality (think cobblestone, wooden
 So for example, a simple implementation would look something like this:
 
 ```java
-public static final RegistryObject<Block> MY_BETTER_BLOCK = BLOCKS.register(
+  public static final Supplier<Block> MY_BETTER_BLOCK = BLOCKS.register(
```

Comment:
```suggestion
  public static final DeferredBlock<Block> MY_BETTER_BLOCK = BLOCKS.register(
```
Should be a `Holder<Block>`, `DeferredHolder<Block, Block>` or a `DeferredBlock<Block>`, whichever you prefer.